idl/partition_checksum.idl.hh

```diff
@@ -103,3 +103,22 @@ enum class repair_row_level_start_status: uint8_t {
 struct repair_row_level_start_response {
     repair_row_level_start_status status;
 };
+
+enum class removenode_cmd : uint32_t {
+     prepare,
+     sync_data,
+     abort,
+     error,
+     ok,
```

Comment:
What is an error command or an ok command?

Reply:
It is used by the response.